utilities/logic/async_wrapper.py

```diff
@@ -170,23 +170,25 @@ def _run_module(wrapped_cmd, jid, job_path):
             sub_pid = os.fork()
             if sub_pid:
                 # the parent stops the process after the time limit
-                remaining = int(time_limit)
+                time_limit = int(time_limit)
 
                 # set the child process group id to kill all children
                 os.setpgid(sub_pid, sub_pid)
 
-                notice("Start watching %s (%s)"%(sub_pid, remaining))
-                time.sleep(step)
```

Comment:
Do not remove this sleep, it is here to allow for the fs to ensure that the required files are in place.
In some cases it will report back to the process everything is OK but then the fork won't see the files to execute yet.